### PR TITLE
REDSHOP-5962: Change css Filter bar of Order Detail & another detail page in Admin

### PR DESCRIPTION
**Pull request by hoangvan149dhv**

### CODE REVIEW

**Comment by ducredweb** [comment]
@hoangvan149dhv  please also make change to `redshop.admin.css`

**Review by ducredweb — DISMISSED** [review]

**Review by anhredweb — DISMISSED** [review]

**Comment by anhredweb** [comment]
@hoangvan149dhv please name PR REDSHOP-XXXX with correct number with JIRA issue.

Thanks.

**Review by anhredweb — DISMISSED** [review]

File: apt-get [hunk 1]
```diff
@@ -0,0 +1 @@
+1
```
Comment (anhredweb) [inline apt-get]: @hoangvan149dhv What is this file? please remove it.

File: media/com_redshop/css/redshop.admin.css [hunk 2]
```diff
@@ -18318,7 +18318,9 @@ i.tools {
 
 #adminForm .filterTool .filterItem {
     display: inline-block;
-    margin: 10px 4px 4px;
+    margin-right: 10px;
+    margin-top: 4px;
+    margin-bottom: 4px;
```
Comment (anhredweb) [inline media/com_redshop/css/redshop.admin.css]: @hoangvan149dhv : please use combined way code:
`margin: 4px 10px 4px 0px;` means: top right bottom left. Please ref here: https://www.w3schools.com/css/css_margin.asp

File: media/com_redshop/css/redshop.admin.min.css [hunk 3]
```diff
@@ -18318,7 +18318,9 @@ i.tools {
 
 #adminForm .filterTool .filterItem {
     display: inline-block;
-    margin: 10px 4px 4px;
```
Comment (anhredweb) [inline media/com_redshop/css/redshop.admin.min.css]: @hoangvan149dhv This is not MIN CSS file. Please make sure this file is minified before using.

**Review by anhredweb — CHANGES_REQUESTED** [review]
@hoangvan149dhv Please update code for change request. Please do not hesitate for asking your colleges if you do not know how to fix.

**Comment by anhredweb** [comment]
@hoangvan149dhv please fix conflict and do minify CSS. If you are not clear please ask backender about those needful changes
Thanks.

**Pull request closed without merging**